docs/MigrationGuide.md

```diff
@@ -93,8 +95,10 @@ To see the biggest differences please consult the [changelog](Changelog.md).
 
 * [#7113](https://github.com/TouK/nussknacker/pull/7113) Scala 2.13 was updated to 2.13.15, you should update your `flink-scala-2.13` to 1.1.2
 
-### Configuration changes
-* [#6979](https://github.com/TouK/nussknacker/pull/6979) Add `type: "activities-panel"` to the `processToolbarConfig` which replaces removed `{ type: "versions-panel" }` `{ type: "comments-panel" }` and `{ type: "attachments-panel" }`
+* [#7187](https://github.com/TouK/nussknacker/pull/7187) JSON decoding in `request` source (request-response processing mode) 
+  and in `kafka` source (streaming processing mode): For small decimal numbers is used either `Integer` or `Long` (depending on number size)
+  instead of `BigDecimal`. This change should be transparent in most cases as this value wes mostly used after `#CONV.toNumber()` invocation
```

Comment:
```suggestion
  instead of `BigDecimal`. This change should be transparent in most cases as this value was mostly used after `#CONV.toNumber()` invocation
```